message/ops.go

```diff
@@ -400,6 +402,8 @@ func ToOp(m *p2p.Message) (Op, error) {
 		return AppRequestOp, nil
 	case *p2p.Message_AppResponse:
 		return AppResponseOp, nil
+	case *p2p.Message_AppError:
+		return AppRequestFailedOp, nil
```

Comment:
Should we change `AppRequestFailedOp` to `AppErrorOp`?

Reply:
renamed